openwisp_controller/config/static/config/js/vpn.js

```diff
@@ -30,9 +25,30 @@ django.jQuery(function ($) {
                 loading.find('p').hide().text(message).fadeIn(250);
             });
         });
+    },
+    toggleIpSubnetFields = function () {
+        // Show IP and Subnet field only for Wireguard backend
+        if ($('#id_backend').val() == 'openwisp_controller.vpn_backends.Wireguard') {
```

Comment:
also when using vxlan, I suggest using include or something smilar to check whether the string contains (case insensitive) wireguard or vxlan, so also custom backends based on wireguard or vxlan which have these words in their path will work

Reply:
another issue: please hide the whole row, otherwise some empty rows are shown:

![Screenshot from 2021-04-13 20-39-32](https://user-images.githubusercontent.com/841044/114641807-6184fe80-9c98-11eb-8f2f-2edda7a2fbd3.png)

Notice the repeated lines, those are empty rows.

Reply:
also, when using wireguard or vxlan, please hide the `ca` and `cert` fields.

Reply:
Fixed these!